automatix/config.py

```diff
@@ -122,6 +122,9 @@ def get_script(args: argparse.Namespace) -> dict:
 
 def collect_vars(script: dict) -> dict:
     var_dict = script.get('vars', {})
+    if var_dict is None:
+        LOG.error('Vars section defined, but empty!\nThis is illegal, either remove the section or add variables.')
```

Comment:
Wenn wirs schon einbauen, vielleicht eher ein warning, dass das möglicherweise nicht richtig ist?

Aber geht für mich auch so in Ordnung.